Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -965,23 +965,6 @@ public function __construct()
         $this->makeTest($expected);
     }
 
-    public function testStatic()
-    {
-        $expected = <<<'EOF'
-<?php
-
-static class Foo
-{
-    static function Foo()
```

Comment:
OK, it's invalid on 4.x too, Thanks @gharlan!

@SpacePossum , please remove it one more time ;)